### PR TITLE
Integrate BrowserConsoleHandler in Symfony

### PR DESCRIPTION
**Pull request by Warxcell**
| Q             | A
| ------------- | ---
| Branch?       | 4.4
| Bug fix?      | yes
| New feature?  | no
| Deprecations? | no
| Tickets       | Fix https://github.com/symfony/monolog-bundle/issues/152
| License       | MIT
| Doc PR        | 

Related PR https://github.com/symfony/monolog-bundle/pull/416

### CODE REVIEW

**Review by derrabus — CHANGES_REQUESTED** [review]
Please add tests.

**Comment by Warxcell** [comment]
Ok, I will wait on https://github.com/Seldaek/monolog/pull/1593, so I can reduce the code and test aswell.

File: src/Symfony/Bridge/Monolog/Handler/BrowserConsoleHandler.php [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+use Monolog\Handler\BrowserConsoleHandler as BaseBrowserConsoleHandler;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpKernel\Event\ResponseEvent;
+use Symfony\Component\HttpKernel\KernelEvents;
+
+class BrowserConsoleHandler extends BaseBrowserConsoleHandler implements EventSubscriberInterface
+{
+    /** @var string */
+    private static $responseFormat;
```
Comment (stof) [inline src/Symfony/Bridge/Monolog/Handler/BrowserConsoleHandler.php:22]: why is this a static property ?

Comment (Warxcell) [reply src/Symfony/Bridge/Monolog/Handler/BrowserConsoleHandler.php:22]: because everything else in this class is static.

**Comment by Warxcell** [comment]
No longer needed.

**Pull request closed without merging**